frontstage/error_handlers.py

```diff
@@ -65,8 +74,8 @@ def connection_error(error):
 
 @app.errorhandler(JWTValidationError)
 def jwt_validation_error(error):
-    logger.error("JWT validation error", url=request.url, status_code=403)
-    return render_template("errors/403-error.html"), 403
+    logger.error(error=error.message, status_code=500)
```

Comment:
This shouldn't be 403, we generate that JWT and hold it, if it is wrong we have a problem in how we generate it. With that in mind no amount of retries will fix this and this should be a 500